pkg/controller/datavolume/upload-controller.go

```diff
@@ -91,6 +97,19 @@ func NewUploadController(
 	return datavolumeController, nil
 }
 
+func (r *UploadReconciler) updatePVCForPopulation(dataVolume *cdiv1.DataVolume, pvc *corev1.PersistentVolumeClaim) error {
+	if dataVolume.Spec.Source.Upload == nil {
```

Comment:
Same as with import.

Reply:
same answer this check is done also in updateAnnotations, wanted to keep it the same